spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/SparseTensor.scala

```diff
@@ -1146,7 +1146,7 @@ object SparseTensor{
       val curLength = currentTensor.nElement()
       val curTensorOffset = currentTensor.storageOffset() - 1
       // copy to concat _values
-      ev.arraycopy(currentTensor.storage().array(), curTensorOffset,
+      System.arraycopy(currentTensor.storage().array(), curTensorOffset,
```

Comment:
why change this?

Reply:
Because `ev.arraycopy(src: Array[T], srcPos: Int, dest: Array[T], destPos: Int, length: Int)` is inherited by  _NumericFloat_ and _NumericDouble_, which requiring Array[Float] or Array[Double].